web/src/app/users/UserPhoneNumberFilterContainer.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { Ref } from 'react'
+import Grid from '@material-ui/core/Grid'
+import { Filter as LabelFilterIcon } from 'mdi-material-ui'
+import FilterContainer from '../util/FilterContainer'
+import TelTextField from '../util/TelTextField'
+import { searchSelector } from '../selectors'
+import { setURLParam } from '../actions'
+import { useDispatch, useSelector } from 'react-redux'
+
+interface UserPhoneNumberFilterContainerProps {
+  anchorRef?: Ref<HTMLElement>
+}
+
+export default function UserPhoneNumberFilterContainer(
+  props: UserPhoneNumberFilterContainerProps,
+): JSX.Element {
+  const searchParam = String(useSelector(searchSelector))
```

Comment:
`searchSelector` always returns a string (default is `''`) so there's no need to cast, however, the types in `url.ts` are a bit ambiguous currently (should probably fix that) so we can just indicate the type for now
```
  const searchParam = useSelector(searchSelector) as string
```

However, a better option would be to use the hook for params directly:
```suggestion
  const [searchParam, setSearchParam] = useURLParam('search', '')
```